app/views/propertyrepresentation/appoint/isThisYourAgent.scala.html

```diff
@@ -22,7 +22,7 @@
 
 @this(govukErrorSummary : GovukErrorSummary, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout)
 
-@(form: Form[Boolean], agentName: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(form: Form[Boolean], agentName: String, agentAddress: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
```

Comment:
should pass the agent details into page. cleans up parameters.

Reply:
agreed